### PR TITLE
Added data class for sports teams

### PR DESCRIPTION
**Pull request by helenjb**
- Added enum classes for Sport and GenderDivision and created a basic data class to represent sports teams

### CODE REVIEW

File: .idea/gradle.xml
Comment (thisjustin123) [inline .idea/gradle.xml:1]: After kelsey merges her PR (ping her if it's not merged), untrack these files (git remove). `.idea` should not be tracked. Her PR already `.gitignore`s them.

File: app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.cornellappdev.android.score.data.models
+
+data class Team(
+    val sport: Sport,
+    val genderDivision: GenderDivision
+)
+enum class Sport {
+    BASEBALL,
+    BASKETBALL,
+    CROSSCOUNTRY,
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt:10]: For any sports that are multiple words, use `CROSS_COUNTRY` naming convention.

File: app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package com.cornellappdev.android.score.data.models
+
+data class Team(
+    val sport: Sport,
+    val genderDivision: GenderDivision
+)
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt:6]: Wait maybe I'm dumb, but what does this class represent? Did I ask you to make this 😅

**Review by thisjustin123 — CHANGES_REQUESTED** [review]
Some questions for you!

File: app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt (hunk 2 above)
Comment (helenjb) [reply app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt:6]: NAURR sorry justin i misinterpreted

File: .idea/gradle.xml
Comment (thisjustin123) [inline .idea/gradle.xml:1]: You still have not git removed this, the `idea/` files are still in the PR.

**Review by thisjustin123 — CHANGES_REQUESTED** [review]
plz try `git rm --cached <file>` against these `.idea/` files to remove them from the PR.

**Review by thisjustin123 — APPROVED** [review]

**Pull request merged**